static/skin/i18n/es-formal.json

```diff
@@ -0,0 +1,90 @@
+{
+	"@metadata": {
+		"authors": [
+			"Camachevaz"
+		]
+	},
+	"name": "Inglés",
```

Comment:
Same problem as in https://github.com/kiwix/kiwix-desktop/pull/1387#discussion_r2309687361

Reply:
whole file has been removed